### DISCUSSION TITLE
Export labeltrees in CSV report

### DISCUSSION THREAD

**Discussion by EmKarr** [biigle/core#575] [Ideas]
Hi,

the CSV report seems quite extensive and lists even the label hierarchy, but lacks information about the labeltree the label is originating from. If this were included, filtering out certain labeltrees during downstream processing (which is a common problem for us) would be much easier. It would be helpful if this could be implemented some time.

Thanks for this really great tool,
Michael

**Comment by mzur** [comment]
Hi, if you want to filter out certain label trees during downstream processing, you can select the "separate label trees" option. This will generate one report file for each label tree and you can discard the files that you don't need.
